tests/api_tests/test_add_documents.py

```diff
@@ -266,11 +276,83 @@ def run():
                 {"d1": "blah"}, {"d2", "some data"}
             ])
             return True
+
         assert run()
 
         args, kwargs = mock__post.call_args
         assert "processes=12" not in kwargs["path"]
 
+    def test_add_documents_deprecated_query_parameters_and_new_api(self):
+        """This test is to ensure that the new API does not accept old query parameters"""
+        self.client.create_index(self.index_name_1)
+        url = f'http://localhost:8882/indexes/{self.index_name_1}/documents?use_existing_tensors=true'  # deprecated query parameter
```

Comment:
It's also worth testing marqo's behavior when given a non-existent query param. For example: `/documents?random_param=true`

Reply:
added